tests/testsuite/cargo_add/rust_version_older/stderr.log

```diff
@@ -0,0 +1,3 @@
+    Updating `dummy-registry` index
+warning: ignoring `rust-version-user` 0.2.1 to satisfy this package's rust-version of 1.70 (use `--ignore-rust-version` to override)
```

Comment:
Huh, just had a thought.  We could specify crates as `rust-version-user@0.2.1`.  That is the command-line syntax.

Thoughts?